Dockerfile.s390x

```diff
@@ -1,4 +1,9 @@
-FROM s390x/alpine:3.10
+ARG QEMU_IMAGE
+FROM ${QEMU_IMAGE} as qemu
+
+FROM s390x/alpine:3.12 as base
```

Comment:
Do we need "as base" here?  Please remove if not.

Reply:
We don't need it and It can be removed.